core/src/main/kotlin/com/malinskiy/marathon/cache/CacheEntryWriter.kt

```diff
@@ -0,0 +1,8 @@
+package com.malinskiy.marathon.cache
+
+import java.io.OutputStream
+
+interface CacheEntryWriter {
+    fun writeTo(output: OutputStream)
```

Comment:
Will it throw exception if there will be error when writing or maybe it should return boolean?

Reply:
Yes, it will throw an exception in case of error